src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -205,7 +205,7 @@ function test_zap() {
     local osd_data=$dir/dir
     $mkdir -p $osd_data
 
-    ${CEPH_DISK} $CEPH_DISK_ARGS zap $osd_data 2>&1 | grep -q 'not full block device' || return 1
+    ${CEPH_DISK} $CEPH_DISK_ARGS zap $osd_data 2>&1 | grep -q 'not full device' || return 1
```

Comment:
@tchaikov 
I changed the grepped messages because the test no longer is only for a block device. 
FreeBSD does not have block devices, so not to "confuse" user, and get useless questions, I thought it was wise to make it more generic. But perhaps I missed some tests.
And things could get worse when goint into theutology...
So perhaps it is easier to just change this back.